global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "8.0.100"
```

Comment:
this can be reverted... needed it for my PC

Reply:
Not a problem